/***************************************************************************
    File                 : globals.h
    Project              : QtiPlot
    --------------------------------------------------------------------
    Copyright            : (C) 2006 by Ion Vasilief, Tilman Hoener zu Siederdissen
    Email (use @ for *)  : ion_vasilief*yahoo.fr, thzs*gmx.net
    Description          : Definition of global constants

 ***************************************************************************/

/***************************************************************************
 *                                                                         *
 *  This program is free software; you can redistribute it and/or modify   *
 *  it under the terms of the GNU General Public License as published by   *
 *  the Free Software Foundation; either version 2 of the License, or      *
 *  (at your option) any later version.                                    *
 *                                                                         *
 *  This program is distributed in the hope that it will be useful,        *
 *  but WITHOUT ANY WARRANTY; without even the implied warranty of         *
 *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the          *
 *  GNU General Public License for more details.                           *
 *                                                                         *
 *   You should have received a copy of the GNU General Public License     *
 *   along with this program; if not, write to the Free Software           *
 *   Foundation, Inc., 51 Franklin Street, Fifth Floor,                    *
 *   Boston, MA  02110-1301  USA                                           *
 *                                                                         *
 ***************************************************************************/

//  Don't forget to change the Doxyfile when changing these!
//! Major version number
const int maj_version = 0;
//! Minor version number (0..9)
const int min_version = 9;
//! Patch version number (0..9)
const int patch_version = 0;
//! Extra version information string (like "alpha", "-2", etc...)
const char * extra_version = "-rc3";
//! Copyright string containing the author names
const char * copyright_string = "Copyright (C) 2004-2007 Ion Vasilief\nCopyright (C) 2006-2007 Tilman Hoener zu Siederdissen, Knut Franke";
//! Release date as a string
const char * release_date = " 2007/06/22";
